src/components/Calculator-Folder/Calculator-childcomponent.js

```diff
@@ -3,6 +3,7 @@ import SharedStateContext from '../../SharedStateContext';
 
 export default function Calculatorchildcomp() {
   const { handleClick, displayValue } = useContext(SharedStateContext);
+  // const Display = displayValue.toString();
```

Comment:
- [ ] [As suggested](https://github.com/stino-x/Math-MAgicians--Microverse-/pull/5#discussion_r1298637701), please make sure to remove the commented code for maintaining a neat and tidy codebase. you can always refer back to old code from git history ✨📌